app/src/main/java/org/sopt/and/presentation/viewmodel/SignInViewModel.kt

```diff
@@ -0,0 +1,42 @@
+package org.sopt.and.presentation.viewmodel
+
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.setValue
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.ViewModelProvider
+
+class SignInViewModel(
+    private val signUpViewModel: SignUpViewModel
+) : ViewModel() {
+
+    var emailLogin by mutableStateOf("")
+    var passwordLogin by mutableStateOf("")
+
+    var emailError by mutableStateOf("")
+    var passwordError by mutableStateOf("")
+
+    fun validate() {
+        emailError = if (emailLogin.isEmpty()) "이메일을 입력하세요." else ""
+        passwordError = if (passwordLogin.isEmpty()) "비밀번호를 입력하세요." else ""
+    }
+
+    fun validateSignIn(email: String, password: String): Boolean {
+        return email == signUpViewModel.email && password == signUpViewModel.password
+    }
```

Comment:
```suggestion
    fun validateSignIn(email: String, password: String) = 
        email == signUpViewModel.email && password == signUpViewModel.password
```
처럼도 표현가능합니다! 더 간결하게 코틀린스럽게 표현하는 함수인데요, 이렇게 쓸수도 있다~ 정도로 알아두시면 좋을 것 같아요 ㅎㅎ